packages/plugins/src/BreadcrumbsPlugin.ts

```diff
@@ -24,7 +24,7 @@ const BreadcrumbsPlugin: PluginType<BreadcrumbsPluginPage, BreadcrumbsPluginOpti
         let currentPage = page;
         let parentDir = path.posix.normalize(path.posix.dirname(currentPage.fullPath));
 
-        while (currentPage !== undefined) {
+        while (currentPage !== undefined && currentPage.fullPath !== '/index.mdx') {
```

Comment:
I remember JP be vocal about not coupling us to specific file types ?
Some of our configs have things like 'extensions' in them, so hardcoding a filetype in source, feels very specific ?
Maybe our position has changed on this but what's the rule going forward and how does this affect config values ?

Reply:
Will rework

Reply:
Fixed